src/pages/Popup/Popup.jsx

```diff
@@ -1,62 +1,67 @@
-import React, { useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
 import logo from '../../assets/img/logo_light.svg';
 import Greetings from '../../containers/Greetings/Greetings';
 import './Popup.css';
 import '../../assets/fontawesome/css/fontawesome.min.css';
 import '../../assets/fontawesome/css/solid.min.css';
 
 const Popup = () => {
+  // Button logic
+  const [buttonRecordActive, setButtonRecordActive] = useState(false);
+  const [buttonPlayActive, setButtonPlayActive] = useState(false);
+  const [recordText, setRecordText] = React.useState("Record");
+  const [recordIconClass, setRecordIconClass] = React.useState("fa-solid fa-circle-dot");
+  const [playText, setPlayText] = React.useState("Play");
+  const [playIconClass, setPlayIconClass] = React.useState("fa-solid fa-circle-play");
+  const buttonRecordClass = (buttonRecordActive) ? 'Button-stop' : 'Button-record';
+  const buttonPlayClass = (buttonPlayActive) ? 'Button-stop' : 'Button-play';
+
+  function recordHandler() {
+    if (buttonRecordActive) {
+      setRecordText("Record");
+      setRecordIconClass("fa-solid fa-circle-dot");
+      setButtonRecordActive(false);
+    }
+    else {
+      if (buttonPlayActive) {
+        const element = document.querySelector('#playButton') || null;
+        if (element) element.click();
```

Comment:
Why are you clicking? this is very odd to do programmaticaly here.

Reply:
I wanted to make sure the other button is not active. Let me do it in other way.